astropy/units/quantity_helper.py

```diff
@@ -636,3 +643,273 @@ def check_output(output, unit, inputs, function=None):
         raise TypeError("Arguments cannot be cast safely to inplace "
                         "output with dtype={0}".format(output.dtype))
     return output
+
+# helpers for ufuncs in scipy.special
+# available ufuncs in this module are at 
+# https://docs.scipy.org/doc/scipy/reference/special.html
+
+try:
+    import scipy.special
```

Comment:
Maybe `from scipy import special as <something-short>` so the text below is shorter as well?

Reply:
Sure, wasn't sure what the preferred style here was.